decisions/mnm_society_decisions.txt

```diff
@@ -2756,64 +2751,106 @@ targeted_decisions = {
 		}
 		
 		allow = {
+			# Warcraft
+			# If you're not in society, you can abduct courtiers only
+			OR = {
+				is_ruler = no
+				FROM = { is_in_society = yes }
+			}
+			
 			in_revolt = no
 			NOT = { war_with = FROM }
 			NOT = { same_society_as = FROM }
 			prisoner = no
-			OR = {
-				AND = {
-					is_landed = no
-					FROM = {
-						has_society_currency_tiny_trigger = yes
+			
+			# Warcraft
+			trigger_if = {
+				limit = {
+					is_ruler = no
+				}
+				trigger_if  = {
+					limit = {
+						FROM = {
+							is_in_society = yes
+							has_society_currency_tiny_trigger = yes
+						}
 					}
+					FROM = { has_society_currency_tiny_trigger = yes }
 				}
-				AND = {
-					is_landed = yes
-					NOT = {
-						higher_tier_than = BARON
+				trigger_else_if = {
+					limit = {
+						FROM = {
+							trait = vampire
+						}
 					}
 					FROM = {
-						has_society_currency_minor_trigger = yes
+						wealth = 25
+						prestige = 50
+						piety = 25
 					}
 				}
-				AND = {
-					higher_tier_than = BARON
-					FROM = {
-						has_society_currency_medium_trigger = yes
-					}
+				trigger_else = {
+					FROM = { has_society_currency_tiny_trigger = yes }
 				}
 			}
+			trigger_else_if = {
+				limit = {
+					lower_tier_than = COUNT
+				}
+				FROM = { has_society_currency_minor_trigger = yes }
+			}
+			trigger_else = {
+				FROM = { has_society_currency_medium_trigger = yes }
+			}
+			
 			is_within_diplo_range = FROM
 		}
 		
 		effect = {
+			# Warcraft
```

Comment:
Also didn't get it here

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/75932611-e9d78400-5e88-11ea-9f40-ca733acadbcf.png)

</details>